pkg/lint/lint_test.go

```diff
@@ -151,3 +153,26 @@ func TestSubChartValuesChart(t *testing.T) {
 		}
 	}
 }
+
+// lint stuck with malformed template object
+// See https://github.com/helm/helm/issues/11391
+func TestMalformedTemplate(t *testing.T) {
+	c := time.After(3 * time.Second)
```

Comment:
why 3s?

Reply:
> why 3s?

Before repair in the code, it will cause infinite loop when lint the malformed template . So  we wait it for 3s to test wheather a  infinite loop happen or not.  

It doesn't have to be 3s. Do you have any better suggestions?

Reply:
@wujunwei ok. got it. Thanks very much.